contents/blog/practical-alerts.mdx

```diff
@@ -0,0 +1,167 @@
+- **Frequency:** set how often you want to check the alert threshold (hourly, daily, weekly, monthly).
+- **Who to notify:** select who should receive an email when the alert is triggered.
+
+<ProductScreenshot
+    imageLight="https://res.cloudinary.com/dmukukwp6/image/upload/filled_alert_627158e6bb.png"
+    imageDark="https://res.cloudinary.com/dmukukwp6/image/upload/alert_people_dark_0ccd7611bd.png"
+    alt="filled alert"
+    classes="rounded"
+/>
+
+Then, click **Create alert** and you'll see the new alert:
+
+<ProductScreenshot
+    imageLight="https://res.cloudinary.com/dmukukwp6/image/upload/insight_alerts_5k_daef47ade1.png"
+    imageDark="https://res.cloudinary.com/dmukukwp6/image/upload/created_alert_dark_1710da74dc.png"
+    alt="insight alerts"
+    classes="rounded"
+/>
+
+For a more detailed walkthrough, you can also check out our [Alerts docs](https://posthog.com/docs/alerts), which we'll update as we add new features.
+
+## Practical uses for alerts
+
+Now that we can create alerts, let's take a look at a handful of practical examples.
+
+### 1. Get an alert when traffic spikes
+
+Want to investigate spikes in traffic to your website? Monitoring the traffic to your website can help you measure the success of marketing campaigns and understand which activities drive the most visitors to your site.
+
+These kinds of alerts can also be useful when traffic spikes are unexpected. We often see this at PostHog when our content gets featured in a popular email newsletter or shared on Hacker News. When we notice an influx of new signups, it can be fun to dig in and try to figure out where they came from.
+
+For example, we can set up an alert when the number of pageviews for our [pricing page](/pricing) rises above 8,000 views for the week. First, we [create a new trend insight](/docs/product-analytics/trends/overview#1-creating-a-new-trend) to track pageviews where "Path Name" equals "/pricing". Then, filter by the year-to-date and group by week.
+
+Then, set up an alert using the steps above to trigger a notification when the number of pageviews rises above 8,000. When the threshold is crossed, this is what the email alert looks like:
+
+![Pricing pageviews alert](https://res.cloudinary.com/dmukukwp6/image/upload/posthog_alert_df58e89828.png)
+
+### 2. Get notified for 404 page anomalies
+
+You can also use alerts to track down errors. For example, you can track the number of visits to your 404 page. If someone visits `posthog.com/some-nonexistent-url`, they will land on our 404 page. Some number of these kinds of visits to this page is to be expected. Anyone that misspells a URL or finds a dead link is bound to land here at some point.
+
+But what about anomalous spikes in 404s? It might mean you deployed a broken link somewhere. Or perhaps you moved a page and need to set up a redirect to prevent users from missing it.
```

Comment:
```suggestion
But what about anomalous spikes in 404s? It might mean you deployed a broken link on a popular page. Or perhaps you moved a page and need to set up a redirect to prevent users from missing it.
```